src/solver.cpp

```diff
@@ -59,8 +59,7 @@ std::vector<Program> PrecompileKernels(const Handle& h, const std::vector<Kernel
     std::vector<Program> programs(kernels.size());
 
     // clang-format off
-    par_for(kernels.size(),
-            max_threads{Value(MIOPEN_COMPILE_PARALLEL_LEVEL{}, 20)},
+    par_for_strided(kernels.size(),
```

Comment:
@ZPEthanWen Can we restore the `MIOPEN_COMPILE_PARALLEL_LEVEL` functionality for debugging/triaging purposes?